app/views/spree/admin/orders/_assemblies.html.erb

```diff
@@ -1,62 +1,54 @@
 <% if order.line_items.assemblies.any? %>
-  <fieldset class="no-border-bottom">
-    <legend align="center" class="" data-hook="">
-      <%= Spree.t(:product_bundles) %>
-    </legend>
-  </fieldset>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h1 class="panel-title">
+        <%= Spree.t(:product_bundles) %>
+      </h1>
+    </div>
+    <table class="table table-bordered product-bundles" data-hook="">
```

Comment:
Looking at the original `table` markup, it only uses `product-bundles` class.
Are the other two classes (`table table-bordered`) really needed?